policyengine_us/variables/gov/states/mt/tax/income/credits/mt_elderly_homeowner_or_renter_credit/mt_elderly_homeowner_or_renter_credit.py

```diff
@@ -0,0 +1,39 @@
+from policyengine_us.model_api import *
+
+
+class mt_elderly_homeowner_or_renter_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana Elderly Homeowner/Renter Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "mt_elderly_homeowner_or_renter_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.mt.tax.income.credits.elderly_homeowner_or_renter_credit
+
+        # Calculate net_household_income
+        standard_exclusion = p.net_household_income.standard_exclusion_amount
+        gross_household_income = tax_unit("mt_gross_household_income", period)
+        reduced_household_income = max_(
+            gross_household_income - standard_exclusion, 0
+        )
+        net_household_income = (
+            p.net_household_income.phase_out_rate.calc(
+                reduced_household_income
+            )
+            * reduced_household_income
+        )
```

Comment:
create a new mt_elderly_homeowner_or_renter_credit_net_household_income variable with the computation:

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/a491b802-8efb-49be-9ca1-38565b24de90)

Reply:
logic is great!